lib/client/components/Location.jsx

```diff
@@ -11,17 +12,18 @@ export default class Location extends React.Component {
   }
 
   handleClick() {
-    navigator.geolocation.getCurrentPosition(
+    this.setState({ location: 'Fetching location...', error: '' });
+
+    Geolocation.getCurrentPosition(
       geo => {
         const lat = geo.coords.latitude;
         const lon = geo.coords.longitude;
 
-        this.setState({ lat, lon, location: `${lat}, ${lon}`, error: '' });
+        this.setState({ lat, lon, location: `${lat.toFixed(4)}, ${lon.toFixed(4)}`, error: '' });
```

Comment:
From what @mjcarroll said, it sounds like this is fine. Long term the coordinates will probably never show, but since it doesn't really adversely affect our ability to get an accurate position I'm fine with it